docs/pages/web/toast.js

```diff
@@ -125,8 +130,12 @@ function ToastExample() {
         </MainSection.Subsection>
 
         <MainSection.Subsection
-          description="When passing in your own Text component for `text`, do not specify `color` on Text. Toast will automatically pick the correct text color for the given `variant`."
-          title="Complex Text"
+          description={`
+The \`text\` prop accepts either a string or [Text](/Text). Use a string for guide toasts without any visual style. Toast will handle the text style and adherence to design guidelines.
+
+If a confirmation toasts text with more complex style is required, such as bold text or inline links, use Text to wrap your message with any additional Text or Link usages contained within. When passing in your own Text component for \`text\`, do not specify \`color\` on Text. Toast will automatically pick the correct text color for the given \`variant\`.
```

Comment:
typo: "a confirmation _toast's_ text with more complex…"

Also: not wanting users to specify `color` on Text sounds like something we should enforce with a lint rule

Reply:
DONE